<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LiuYongjian" ignored="0" name="Surefire test" tests="1" failures="1" timestamp="17 二月 2019 09:30:42 GMT" time="22.834" errors="0">
  <testcase name="runScenario" time="21.782" classname="cucumber.api.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="URL judgement has been failed!">
      <![CDATA[java.lang.AssertionError: URL judgement has been failed!
at com.frank.cucumberframework.stepdefinition.web.LoginTestSteps.enterIntoDashboard(LoginTestSteps.java:68)
at ✽.redirect to dashboard page(src/test/resources/features/web_login.feature:7)
... Removed 1 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Surefire test -->
